pages/tags/[tagId].js

```diff
@@ -0,0 +1,68 @@
+import Head from 'next/head';
```

Comment:
Maybe not needed, maybe needed.

It should be easy to add this file later when we need it, yeah?

As such, please remove this file.

Please only include files in this commit for things that we will 100% definitely need.